packages/primero/src/Adaptor.js

```diff
@@ -176,9 +176,19 @@ function cleanupState(state) {
  *   remote: true,
  *   case_id: "6aeaa66a-5a92-4ff5-bf7a-e59cde07eaaz",
  * });
+ * @example <caption>Get case from Primero with referrals</caption>
```

Comment:
This should say "get cases" (plural), no? Maybe "Get cases and their referrals" is better

Reply:
Fixed